### PR TITLE
Support onetrueliteral (and new width specifier).

### PR DESCRIPTION
**Pull request by ucbjrl**
This should be merged when ucb-bar/chisel3#372 is merged.

### CODE REVIEW

**Comment by ducky64** [comment]
It seems there's still a few deprecated methods in use. Try compiling with deprecation warnings by adding

`scalacOptions := Seq("-deprecation")`

to build.sbt. I think the regex replacements can automatically fix many of the remaining deprecations.

Also note that the width specifier is new so none of this can be used until the corresponding chisel3 branch is merged.

**Comment by ucbjrl** [comment]
Right. I think it would be a good idea to add the width specifier to
chisel3 master first, so we can migrate code while the old methods still
exist.

File: src/test/scala/examples/Adder.scala [hunk 1]
```diff
@@ -45,13 +45,13 @@ class AdderExerciser extends Exerciser {
     val count = 20 // this forces ranges to all be the same size
     Range(start, start + count)
   }
-  val in0_vec = Vec(range(x_range_start).map(UInt(_)))
-  val in1_vec = Vec(range(y_range_start).map(UInt(_)))
+  val in0_vec = Vec(range(x_range_start).map((_).asUInt()))
```
Comment (ducky64) [inline src/test/scala/examples/Adder.scala]: I don't think you need the parentheses, you should just be able to write `_.asUInt`

File: src/main/scala/chisel3/iotesters/SteppedHWIOTester.scala [hunk 2]
```diff
@@ -132,30 +132,30 @@ abstract class SteppedHWIOTester extends HWIOTester {
...
     val ok_to_test_output_values = Vec(
       test_actions.map { step =>
-        Bool(step.output_map.contains(output_port))
+        (step.output_map.contains(output_port)).asBool
       }
     )
 
     when(ok_to_test_output_values(counter.value)) {
-      when(output_port.toBits() === output_values(counter.value).toBits()) {
+      when(output_port.asUInt() === output_values(counter.value).asUInt()) {
```
Comment (ducky64) [inline src/main/scala/chisel3/iotesters/SteppedHWIOTester.scala]: I don't think we need the empty parentheses for asUInt, just use like `output_port.asUInt`?

File: src/test/scala/examples/Adder.scala [hunk 3]
```diff
@@ -45,13 +45,13 @@ class AdderExerciser extends Exerciser {
     val count = 20 // this forces ranges to all be the same size
     Range(start, start + count)
   }
-  val in0_vec = Vec(range(x_range_start).map((_).asUInt()))
-  val in1_vec = Vec(range(y_range_start).map((_).asUInt()))
+  val in0_vec = Vec(range(x_range_start).map((_).asUInt))
+  val in1_vec = Vec(range(y_range_start).map((_).asUInt))
```
Comment (ducky64) [inline src/test/scala/examples/Adder.scala]: You should just be able to do `_.asUInt`

**Comment by ducky64** [comment]
lgtm

**Pull request merged**